ktor-server/ktor-server-plugins/ktor-server-metrics-micrometer/jvm/src/io/ktor/server/metrics/micrometer/MicrometerMetrics.kt

```diff
@@ -119,6 +119,34 @@ public class MicrometerMetricsConfig {
     public fun timers(block: Timer.Builder.(ApplicationCall, Throwable?) -> Unit) {
         timerBuilder = block
     }
+
+    internal var transformRoute: RoutingNode.() -> String = { path }
+
+    /**
+     * Configures mapping function for the route label string of the [CallMeasure].
+     * Defaults to [RoutingNode.path].
+     *
+     * **Examples:**
+     *
+     * Use the toString() function of the RoutingNode:
+     * ```kotlin
+     * transformRoute {
+     *    toString()
+     * }
+     * ```
+     *
+     * Remove a prefix from the path:
+     * ```kotlin
+     * transformRoute {
+     *     path.removePrefix("/path/prefix")
+     * }
+     * ```
+     *
+     * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.server.metrics.micrometer.MicrometerMetricsConfig.transformRoute)
+     */
+    public fun transformRoute(block : RoutingNode.() -> String) {
```

Comment:
I'd make `RoutingNode` a parameter as it is a common pattern for transforming functions.
```suggestion
    public fun transformRoute(block: (RoutingNode) -> String) {
```